src/__stories__/config.js

```diff
@@ -0,0 +1,8 @@
+const { configure } = require("@kadira/storybook")
+
+function loadStories() {
+  require("../components/__stories__/artwork")
+  require("../components/__stories__/artwork_grid")
+}
+
+configure(loadStories, module);
```

Comment:
This file has to exist _inside_ the FuseBox `homeDir`, trying to recurse from outside of it with a relative path does not seem to make FuseBox happy.